site/content/en/latest/tasks/traffic/local-rate-limit.md

```diff
@@ -245,6 +245,188 @@ server: envoy
 
 ```
 
+## Rate Limit Specific User Unless with Admin Role
+
+Here is an example of a rate limit implemented by the application developer to limit a specific user by matching on a custom `x-user-id` header
+with a value set to `one`. But the user must not be limited if logging in with admin role determined by custom header `x-user-role` set to `admin`.
+
+{{< tabpane text=true >}}
+{{% tab header="Apply from stdin" %}}
+
+```shell
+cat <<EOF | kubectl apply -f -
+apiVersion: gateway.envoyproxy.io/v1alpha1
+kind: BackendTrafficPolicy 
+metadata:
+  name: policy-httproute
+spec:
+  targetRefs:
+  - group: gateway.networking.k8s.io
+    kind: HTTPRoute
+    name: http-ratelimit
+  rateLimit:
+    type: Local
+    local:
+      rules:
+      - clientSelectors:
+        - headers:
+          - name: x-user-id
```

Comment:
NIt: typically, user's role is tied to a specific user id. Could we change the first header in this example to something like "x-user-org"?

like:
```
   - name: x-user-org
    value: foo
    - name: x-user-role
    ....
```

Reply:
Agree, changed the example to not rate limit if `x-ord-id` is set to `test`.